tinygrad/runtime/ops_gpu.py

```diff
@@ -48,6 +48,7 @@ def synchronize(self):
 class CLBuffer(RawBufferCopyInOut, RawBufferTransfer):
   def __init__(self, size, dtype, device='0'): super().__init__(size, dtype, allocator=CL.cl_allocator, **{'device': device})
   def _copyin(self, x:np.ndarray):
+    x = np.require(x, requirements=['A'])
```

Comment:
can the align requirement just be arrayed with the contiguous requirement below?

Reply:
Contiguous did not work. I forget to mention, should we consider aligning all numpy arrays in RawBufferCopyIn? Updating the original comment as well.

Reply:
I meant like does this have to be a seperate line, can the 'A' just be put in the same requirements as the 'C' below

Reply:
oh lol, yes my fault.